packages/dataviews/src/style.scss

```diff
@@ -411,6 +412,18 @@
 			}
 		}
 
+		.dataviews-item-actions .components-button {
```

Comment:
Make the actions buttons (primary action + all actions) hidden, unless is hovered/focused/selected.